xml/docu_styleguide.taglist.xml

```diff
@@ -0,0 +1,1175 @@
+       </screen>
+      </entry>
+      <entry>
+       Name of a software or application package
+      </entry>
+      <entry>No</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+parameter
+       </screen>
+      </entry>
+      <entry>
+       A value or a symbolic reference to a value
+      </entry>
+      <entry>No</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+phrase
+       </screen>
+      </entry>
+      <entry>
+       A span of text
+      </entry>
+      <entry>Yes</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+productname
+       </screen>
+      </entry>
+      <entry>
+       <para>
+       The formal name of a product
+       When referring to an application, add a
+       &lt;phrase role="productname"/> element around it, for example:
+       </para>
+       <screen>
+&lt;phrase role="productname">
+LibreOffice&lt;/phrase> is an
+office suite.
+       </screen>
+       </entry>
+      <entry>No</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+prompt
+       </screen>
+      </entry>
+      <entry>
+       A character or string indicating the start of an input field in a
+       computer display
+      </entry>
+      <entry>No</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+quote
+       </screen>
+      </entry>
+      <entry>
+       To quote from sources, such as books. In all other cases, do not use
+       quotation marks.
+      </entry>
+      <entry>Yes</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+replaceable
+       </screen>
+      </entry>
+      <entry>
+       Content that may or must be replaced by the user
+      </entry>
+      <entry>No</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+systemitem
+       </screen>
+      </entry>
+      <entry>
+       A system-related item or term
+      </entry>
+      <entry>No</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+subscript
+       </screen>
+      </entry>
+      <entry>
+       <para>A subscript. For example:</para>
+       <para>H<subscript>2</subscript>O</para>
+      </entry>
+      <entry>Yes</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+superscript
+       </screen>
+      </entry>
+      <entry>
+       <para>A superscript. For example:</para>
+       <para>X<superscript>2</superscript></para>
+      </entry>
+      <entry>Yes</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+tag
+       </screen>
+      </entry>
+      <entry>
+       A component of XML (or SGML) markup
+      </entry>
+      <entry>No</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+trademark
+       </screen>
+      </entry>
+      <entry>
+       To mark a trademark with <trademark>TM</trademark>
+      </entry>
+      <entry>No</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+uri
+       </screen>
+      </entry>
+      <entry>
+       For links that should not be clickable
```

Comment:
The element `<uri>` is for more than just links. URI, the acronym, stands for _Uniform Resource Identifier_. It's a more general term which includes links, but also some kind of general identifier like namespaces, for example `urn:oasis:names:tc:entity:xmlns:xml:catalog`.

Maybe something like this:

```suggestion
       For general identifiers (like links) that should not be clickable
```